### ISSUE TITLE
get nothing in `getRecentMessages`

### ISSUE THREAD

**Issue by ghost** [CYBAI/jsline-api#2]
This api is inspired and partially based on carpedm20's work
But as it seems the `getRecentMessages` method is malfunctioning in that api due to some change in the line protocol
Is the same problem occurring in your api?

**Comment by CYBAI** [comment]
@jackzett Yes, I also have the same issue in my api. Still considering how to fix that.
[carpedm20/LINE#71](https://github.com/carpedm20/LINE/issues/71)
[carpedm20/LINE#74](https://github.com/carpedm20/LINE/issues/74)

**Comment by ghost** [comment]
Okay thnx for letting me know
